src/sentry/models/servicehook.py

```diff
@@ -111,3 +111,14 @@ def add_project(self, project):
             project_id=project.id,
             service_hook_id=self.id,
         )
+
+    def get_projects(self):
+        """
+        Get all projects associated with a service hook.
+
+        """
+        ids = ServiceHookProject.objects.filter(
+            service_hook_id=self.id,
+        ).values_list('project_id', flat=True)
+
+        return Project.objects.filter(id__in=ids)
```

Comment:
This will do two queries to get the projects. Is there a way to get them with a join, or will Django not make that easy because the two models don't have a formal relationship defined?

Reply:
honestly right now I don't even really need the project instances. I thought I would need them to check against the project to make sure we don't send webhook events for projects people didn't give us permission too, but right now we don't even let you pick which projects. 

I'm going to just return the list of ids for now, and we can revisit when we add in logic for limiting permissions by project